engine.go

```diff
@@ -8,25 +8,29 @@ import (
 	"gopkg.in/src-d/go-mysql-server.v0/sql/parse"
 )
 
+const mysqlVersion = "8.0.11"
+const defaultVersionPostfix = "go-mysql-server"
+
 // Engine is a SQL engine.
 type Engine struct {
 	Catalog  *sql.Catalog
 	Analyzer *analyzer.Analyzer
 }
 
 // New creates a new Engine
-func New(c *sql.Catalog, a *analyzer.Analyzer) *Engine {
+func New(c *sql.Catalog, a *analyzer.Analyzer, versionPostfix string) *Engine {
 	c.RegisterFunctions(function.Defaults)
+	c.RegisterFunction("version", sql.FunctionN(function.NewVersion(mysqlVersion+"-"+versionPostfix)))
```

Comment:
`mysqlVersion` should probably always be used, an only postfix be given to `NewVersion`. Or make `mysqlVersion` public, since its `go-mysql-version` the one that implements MySQL stuff and controls supported features.

Reply:
Could you please help me to understand which part of `go-mysql-version` defines mysql version?

Originally my reasoning was `engine` controls supported features. So I could create `my-engine` and add more features from newer version. That's why I put constant here.
Do you think it should be defined in `Version` UDF itself? Or any other place?

Reply:
`mysqlVersion` should be inside the `Version` function. So at the end you only need to send to the constructor `versionPostfix`: